components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -66,15 +83,20 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
 
   // Form info
   public expressionForm: FormGroup;
+  public expressionOutput: string;
+  private formSubscription: Subscription;
+  public allIdentity: string;
+
 
   constructor(
     private store: Store<NgrxStateAtom>,
     private router: Router,
     fb: FormBuilder) {
 
     this.expressionForm = fb.group({
-      // Must stay in sync with error checks in policy-add-members.component.html
-      expression: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
+      type: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]],
+      identity: [''],
+      name: ['']
```

Comment:
We start the second two form elements off with no validation, so that we can conditionally add/remove validations later on.